groups/sig-node/groups.yaml

```diff
@@ -32,6 +32,29 @@ groups:
       - mpatel@redhat.com
       - s.kanzhelev@gmail.com
 
+  - email-id: sig-node@kubernetes.io
```

Comment:
We will need one for CI group. Should I send a separate PR it you can help with it?

Reply:
I'd do that separately as a follow up :+1: 
Hoping to get this merged relatively quickly